src/libpmemobj/pmalloc.c

```diff
@@ -282,15 +282,10 @@ pmalloc_recovery(PMEMobjpool *pop, void *data, unsigned length)
 	struct lane_alloc_layout *sec = data;
 	ASSERT(sizeof(*sec) <= length);
 
-<<<<<<< HEAD
```

Comment:
Each change from this commit should be squashed to the relevant commit. Current state is unbisectable.